docs/Running Migrations.md

```diff
@@ -26,30 +26,31 @@ defmodule MyApp.ReleaseTasks do
     :postgrex,
     :ecto
   ]
-  
-  @myapps [
-    :myapp
-  ]
 
-  @repos [
-    MyApp.Repo
-  ]
+  def myapp do
+    {:ok, app} = :application.get_application(__MODULE__)
```

Comment:
Can you make this `Application.get_application(__MODULE__)` for consistency?

Reply:
applied